tikv/closure_exec.go

```diff
@@ -0,0 +1,162 @@
+package tikv
+
+import (
+	"bytes"
+	"math"
+
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/types"
+	"github.com/juju/errors"
+	"github.com/pingcap/tipb/go-tipb"
+	"github.com/pingcap/tidb/util/codec"
+)
+
+// tryBuildClosureExecutor tries to build a closureExecutor for the DAGRequest.
+// currently, only 'count(*)' is supported, but we can support all kinds of requests in the future.
+func (svr *Server) tryBuildClosureExecutor(dagCtx *dagContext, dagReq *tipb.DAGRequest) *closureExecutor {
+	executors := dagReq.Executors
+	if len(executors) != 2 {
```

Comment:
Please add comments why compares to 2?

Reply:
This will be removed after we support simple IndexScan and TableScan.